packages/edit-post/src/components/sidebar/post-schedule/index.js

```diff
@@ -3,21 +3,31 @@
  */
 import { __, sprintf } from '@wordpress/i18n';
 import { PanelRow, Dropdown, Button } from '@wordpress/components';
-import { useRef } from '@wordpress/element';
+import { useCallback, useState } from '@wordpress/element';
 import {
 	PostSchedule as PostScheduleForm,
 	PostScheduleCheck,
 	usePostScheduleLabel,
 } from '@wordpress/editor';
 
 export default function PostSchedule() {
-	const anchorRef = useRef();
+	// Use internal state instead of a ref to make sure that the component
+	// re-renders when then anchor's ref updates.
+	const [ popoverAnchor, setPopoverAnchor ] = useState();
+	const rowCallbackRef = useCallback( ( node ) => {
```

Comment:
Same as https://github.com/WordPress/gutenberg/pull/43799#discussion_r962960103

Everything looks good otherwise, though!

Reply:
Done in https://github.com/WordPress/gutenberg/pull/43808/commits/84c7861bb63e880f761bf1eeadeb7cec3f50a8e0